Code/Engine/Foundation/IO/FileSystem/Implementation/FileSystem.cpp

```diff
@@ -702,16 +702,36 @@ ezResult ezFileSystem::FindFolderWithSubPath(const char* szStartDirectory, const
   result = szStartDirectory;
   result.MakeCleanPath();
 
-  ezStringBuilder FullPath;
+  ezStringBuilder FullPath, sRedirection;
 
   while (!result.IsEmpty())
   {
     FullPath = sStartDirAbs;
+    FullPath.AppendPath("ezSdkRoot.txt");
```

Comment:
I think the name of this file will be configurable later, as it is currently hacked into this function.